### ISSUE TITLE
Support scrolling CTkOptionMenu with mouse wheel

### ISSUE THREAD

**Issue by HenestrosaDev** [TomSchimansky/CustomTkinter#2135]
There is no way to configure this at this time. The user has to click the up/down arrows if the options list doesn't fit the screen height. I think it would be nice to allow scrolling with the mouse wheel for a more friendly UX.

**Comment by Akascape** [comment]
@HenestrosaDev That's why I made CtkScrollableDropdown widget: https://github.com/Akascape/CTkScrollableDropdown

**Comment by HenestrosaDev** [comment]
@Akascape Yeah, I was looking for something like this! I wish CustomTkinter would add it to the package. I've added it to my [Audiotext](https://github.com/HenestrosaDev/audiotext) project and credited your package. Thank you!

**Comment by Akascape** [comment]
@HenestrosaDev This type of dropdown was there in the earlier version of customtkinter. But there were issues with some macosx versions. But I redesigned the whole dropdown, fixing some major bugs, then uploaded the package for everyone. Thanks for using it in your project and giving me credits.

There are many more ctk widget project available in my account, you may also find them helpful.
